README.md

```diff
@@ -54,9 +54,13 @@ Requires the following to be installed on the deployment host:
 - [netaddr](https://github.com/netaddr/netaddr)
 - [skopeo](https://github.com/containers/skopeo)
 - [podman](https://github.com/containers/podman/)
+- [kubectl + oc](https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html)
+
+**Important Note** `openshift-clients` requires an [activation through the Red Hat Subscription Manager](https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-rpm_cli-developer-commands). It is required for the post-install verification steps.
```

Comment:
The official name is RHSM (Red Hat Subscription Management) https://access.redhat.com/products/red-hat-subscription-management